vm/src/utils.rs

```diff
@@ -48,13 +51,57 @@ pub fn from_relocatable_to_indexes(relocatable: Relocatable) -> (usize, usize) {
     }
 }
 
+pub fn felt_to_biguint(felt: crate::Felt252) -> BigUint {
+    let big_digits = felt
+        .to_le_digits()
+        .into_iter()
+        .flat_map(|limb| [limb as u32, (limb >> 32) as u32])
+        .collect();
+    BigUint::new(big_digits)
+}
+
+pub fn felt_to_bigint(felt: crate::Felt252) -> BigInt {
+    felt_to_biguint(felt).to_bigint().unwrap()
+}
+
+pub fn biguint_to_felt(biguint: &BigUint) -> Result<crate::Felt252, MathError> {
+    // TODO This funtions should return a Felt252 instead of a Result
+    Ok(crate::Felt252::from_bytes_le_slice(&biguint.to_bytes_le()))
+}
+
+pub fn bigint_to_felt(bigint: &BigInt) -> Result<crate::Felt252, MathError> {
+    let (sign, bytes) = bigint
+        .mod_floor(&CAIRO_PRIME.to_bigint().unwrap())
+        .to_bytes_le();
+    let felt = crate::Felt252::from_bytes_le_slice(&bytes);
+    if sign == Sign::Minus {
+        Ok(felt.neg())
+    } else {
+        Ok(felt)
+    }
+}
```

Comment:
The new version of `starknet-core-types` implements conversions between Felt & biguint/bigint so we can use those and remove these functions